easybuild/easyconfigs/w/Wannier90/Wannier90-2.0.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'MakeCp'
+
+name = 'Wannier90'
+version = '2.0.1'
+
+homepage = 'http://www.wannier.org'
+description = """A tool for obtaining maximally-localised Wannier functions"""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.wannier.org/code']
+
+# ugly hack
+prebuildopts = 'rm make.sys; touch make.sys; '
+prebuildopts += 'F90=ifort COMMS=mpi MPIF90=mpiifort FCOPTS="-O2" LDOPTS="-O2" LIBDIR=$LAPACK_LIB_DIR LIBS=$LIBLAPACK '
```

Comment:
regarding the difference between the easyconfigs, wannier 2.x supports MPI, 1.x doesn't (however, some applications still only support 1.x; 1.2 is the last release of the 1.x series, 2.0.1 is the latest releast of 2.x)

regarding LIBLAPACK quoting, will do, thanks!

regarding the optimizations flags, of course, sorry!